packages/eslint-plugin-orbit-components/babel.config.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+const browserslist = require("@kiwicom/browserslist-config");
+
+module.exports = {
+  presets: [
+    [
+      "@babel/preset-env",
+      {
+        targets: {
+          node: true,
+          browsers: browserslist,
```

Comment:
This code won't be running in the browser, so it's not necessary to convert it for browsers. I recommend adding a `.browserslistrc` file in this workspace containing `current node`. That way `@babel/preset-env` should automatically detect it and use it, and hopefully not merge that config with our root `.browserslistrc` file 🤞 But it doesn't matter if that happens, I'll fix it later.